.eslintrc.yaml

```diff
@@ -101,3 +101,4 @@ rules:
   'lines-between-class-members': 0
   'react-hooks/rules-of-hooks': 'error'
   'react-hooks/exhaustive-deps': 'warn'
+  'max-classes-per-file': 0
```

Comment:
I guess this was added in the latest version...

Reply:
Yeah. I had this in the mc-be too. I just added it to the files having multiple classes so we create more of them. If you don't care about the rule disabling it entirely is sound obviously.

Reply:
Yeah doing it on a case by case basis might be better